node/text-generation/README.md

```diff
@@ -0,0 +1,75 @@
+# 🤖 Node Text Generation Function
+
+Ask question, and use Huggingface inference models to get the answer.
+
+## 🧰 Usage
+
+### GET /
+
+HTML form for interacting with the function.
+
+### POST /
+
+Query the model for a completion.
+
+**Parameters**
+
+| Name         | Description                          | Location | Type               | Sample Value                  |
+| ------------ | ------------------------------------ | -------- | ------------------ | ----------------------------- |
+| Content-Type | The content type of the request body | Header   | `application/json` | N/A                           |
+| prompt       | Text to prompt the model             | Body     | String             | `Write a haiku about Mondays` |
+
+Sample `200` Response:
+
+Response from the model.
+
+```json
+{
+    "ok": true,
+    "completion": "Monday's heavy weight, Dawning with a sigh of grey, Hopeful hearts await."
+}
+```
```

Comment:
Is there a way to specify the length of the response? If yes, it would be good to add it as a parameter